common/persistence/dataInterfaces.go

```diff
@@ -605,6 +606,15 @@ type (
 		ScheduledID         int64
 	}
 
+	// FailoverMarkerTask is the marker for graceful failover
+	FailoverMarkerTask struct {
+		TaskID              int64
+		VisibilityTimestamp time.Time
+		Version             int64
+		SourceCluster       string
```

Comment:
Does the underlying DB schema has a field for the `sourceCluster`?

Reply:
No. We don't have. Currently, I just re-use a string field.

Reply:
I see. Probably add some comments then?

Reply:
I will update the schema in the following PR.